report-viewer/src/views/Overview.vue

```diff
@@ -0,0 +1,235 @@
+        </TextInformation>
+        <TextInformation label="Date of execution" :value="overview.dateOfExecution"/>
+        <TextInformation label="Duration (in ms)" :value="overview.durationOfExecution"/>
+      </div>
+      <div id="logo-section">
+        <img id="logo" src="@/assets/logo-nobg.png" alt="JPlag">
+      </div>
+    </div>
+
+      <div class="column-container" style="width: 35%">
+        <div id="metrics">
+          <p class="section-title">Metric:</p>
+          <div id="metrics-list">
+            <MetricButton v-for="(metric, i) in overview.metrics" :key="metric.metricName"
+                          :id="metric.metricName"
+                          :metric-threshold="metric.metricThreshold"
+                          :metric-name="metric.metricName"
+                          :is-selected="selectedMetric[i]"
+                          @click="selectMetric(i)"/>
+          </div>
+        </div>
+        <p class="section-title">Distribution:</p>
+        <DistributionDiagram class="full-width" :distribution="distributions[selectedMetricIndex]"/>
+      </div>
+    <div class="column-container" style="width: 35%">
+        <p class="section-title">Top Comparisons:</p>
+        <div id="comparisonsList">
+          <ComparisonsTable :top-comparisons="topComps[selectedMetricIndex]" :clusters="overview.clusters"/>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import {defineComponent, ref} from "vue";
+import store from "@/store/store";
+import router from "@/router";
+import TextInformation from "../components/TextInformation";
+import DistributionDiagram from "@/components/DistributionDiagram";
+import MetricButton from "@/components/MetricButton";
+import ComparisonsTable from "@/components/ComparisonsTable";
+import {OverviewFactory} from "@/model/factories/OverviewFactory";
+import IDsList from "@/components/IDsList";
+
+export default defineComponent({
+  name: "Overview",
+  components: {IDsList, ComparisonsTable, DistributionDiagram, MetricButton, TextInformation },
+  setup() {
+    let overview;
+    //Gets the overview file based on the used mode (zip, local, single).
+    if (store.state.local) {
+      try {
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        overview = OverviewFactory.getOverview(require("../files/overview.json"))
+      } catch (e) {
+        router.back()
+      }
+    } else if (store.state.zip) {
+      overview = OverviewFactory.getOverview(JSON.parse(store.state.files["overview.json"]))
+    } else if (store.state.single) {
+      overview = OverviewFactory.getOverview(JSON.parse(store.state.fileString))
+    }
+
+    /**
+     * Handles the selection of an Id to anonymize.
+     * If all submission ids are provided as parameter it hides or displays them based on their previous state.
+     * If a single id is provided it hides all of the other ids except for the chosen one.
+     * @param id
+     */
+    const handleId = (id) => {
```

Comment:
The representation of anonymized/non-anonymized IDs is still missing.

Reply:
@sebinside Can you provide some more detail?

Reply:
@cholakov11 I meant that one can click on a submission in the ID list to anonymize all others, but there is no visible feedback which are currently shown. E.g., in the attached image, one cannot see that only A is selected
![image](https://user-images.githubusercontent.com/7331248/154985430-11a0d711-b482-4a35-b9bb-71587be25c73.png)